pilot/cmd/pilot-agent/status/ready/probe_test.go

```diff
@@ -35,7 +35,7 @@ func TestEnvoyStatsCompleteAndSuccessful(t *testing.T) {
 
 	server := createAndStartServer(liveServerStats)
 	defer server.Close()
-	probe := Probe{LocalHostAddr: "localhost", AdminPort: 1234}
```

Comment:
Request HTTP without host is not allowed in HTTP1.1. I added them in some test cases, but encountered more empty host requests in test cases with updates from master branch. So I wrote a check in 'stats.go'(in the next diff window) and removed the redundant configurations.